.github/workflows/run-adaptivity-test.yml

```diff
@@ -2,7 +2,8 @@ name: Run tests for adaptivity
 on:
   push:
     branches:
-      - "*"
+      - main
+      - develop
```

Comment:
Would suggest pushing some changes via other PRs (or direct commits to develop, if the change is straightforward), rather than part of a PR where this is not directly relevant.